code/modules/goonhub/api/models/medals/MedalResource.dm

```diff
@@ -0,0 +1,25 @@
+
+/// MedalResource
+/datum/apiModel/Tracked/MedalResource
+	var/title				= null // string
+	var/description	= null // string
```

Comment:
What about the images?

Reply:
Image isn't a stored column. It's a URL derived from the medal ID. e.g. `/storage/medals/1.png`